keyserver/server.go

```diff
@@ -38,7 +42,8 @@ type ConiksServer struct {
 	stop     chan struct{}
 	waitStop sync.WaitGroup
 
-	secretKey crypto.SigningKey
+	db        kv.DB
+	secretKey sign.PrivateKey
```

Comment:
Are we going to keep the server's secret key in here for now, and move it to the STR policies as we were discussing in https://github.com/coniks-sys/coniks-go/pull/39#discussion_r73080612, once this pull is merged/when we finalize #47?